src/impl/vamp/robots/baxter/fk.hh

```diff
@@ -15388,6 +15388,10 @@ namespace vamp::robots::baxter
 
         return true;
     }
+
+    inline auto eefk(const std::array<float, 14> &q) noexcept -> std::array<float, 7>
+    {
```

Comment:
Prefer to throw an exception? Or return some known value; shouldn't rely on failure to return, just in case.

Reply:
Really the binding helper struct should conditionally include methods based on what the robot in question provides - not hard to do with `std::enable_if`.

Reply:
Let's throw an exception for now - better to have it explode loudly.